pkg/downloader/manager.go

```diff
@@ -371,6 +371,10 @@ func (m *Manager) getRepoNames(deps []*chartutil.Dependency) (map[string]string,
 	// by Helm.
 	missing := []string{}
 	for _, dd := range deps {
+		if dd.Repository == "" {
+			errorMessage := fmt.Sprintf("no 'repository' field specified for dependency: %q", dd.Name)
+			return nil, errors.New(errorMessage)
```

Comment:
This can be streamlined a little:

```
return nil, fmt.Errorf("no 'repository' field specified for dependency: %q", dd.Name)
```

Reply:
Thank you for the review. I have updated the PR.